Sprint-1/interpret/percentage-change.js

```diff
@@ -20,3 +20,17 @@ console.log(`The percentage change is ${percentageChange}`);
 // d) Identify all the lines that are variable declarations
 
 // e) Describe what the expression Number(carPrice.replaceAll(",","")) is doing - what is the purpose of this expression?
+
+
+// a) 1-carPrice = Number(carPrice.replaceAll(",", ""));   2-priceAfterOneYear = Number(priceAfterOneYear.replaceAll("," ""));
+//       3- console.log(`The percentage change is ${percentageChange}`);
+
+// b) there should be a ',' between the argument and the replacement
+// replaceAll is not a function, you should use replace instead
+
+// c) 1- carPrice   2-priceDifference
```

Comment:
These are not quite the "lines" with the declarations.

Reply:
fixed